static/js/src/dynamic-forms.js

```diff
@@ -194,6 +193,25 @@ import { prepareInputFields } from "./prepare-form-inputs.js";
       }
     }
 
+    function setDataLayerConsentInfo() {
+      const modalForm = document.querySelector(".js-modal-form");
```

Comment:
I had a look through [our modal forms](https://github.com/canonical/ubuntu.com/tree/main/templates/shared/forms/interactive) and not all of them have the class name `js-modal-form`, we will have to use a different query or add it to them